### PR TITLE
Box create attribute & modification entity name

### PR DESCRIPTION
**Pull request by benKazmierczak**
This PR contains the box allowing the addition of attributes (html css only)
The js of modification of the name of the entity
Thx for your feedback

### CODE REVIEW

File: assets/js/generator/Generator.js [hunk 1]
```diff
@@ -40,4 +40,42 @@ class Generator {
...
+document.addEventListener("DOMContentLoaded", function() {
+    var elems = document.querySelectorAll("select");
+    var instances = M.FormSelect.init(elems);
+});
+
+/**
+ * @description: Display a box with additional data
+ */
+
+let btnBox = document.getElementById('add-attr_');
```
Comment (mehdou92) [inline assets/js/generator/Generator.js]: Const variable in js if the variable is not modified 💯

File: assets/js/generator/Generator.js [hunk 2]
```diff
@@ -40,4 +40,42 @@ class Generator {
     }
 }
 
+document.addEventListener("DOMContentLoaded", function() {
+    var elems = document.querySelectorAll(".collapsible");
```
Comment (mehdou92) [inline assets/js/generator/Generator.js]: Const or let here 👍

**Review by mehdou92 — CHANGES_REQUESTED** [review]
Little change in js, for the status of the variable ty benji

File: assets/js/generator/Generator.js (hunk 1 above)
Comment (benKazmierczak) [reply assets/js/generator/Generator.js]: changed to a const

File: assets/js/generator/Generator.js (hunk 2 above)
Comment (benKazmierczak) [reply assets/js/generator/Generator.js]: changed to a let

**Review by Backins — APPROVED** [review]
Ok for me

**Review by mehdou92 — APPROVED** [review]

**Review by jeremNoel — APPROVED** [review]

**Pull request merged**